.gitlab/pkg_metrics.yml

```diff
@@ -88,6 +88,7 @@ send_pkg_size-a7:
     - agent_deb-x64-a7
     - iot_agent_deb-x64
     - dogstatsd_deb-x64
+  #  - heroku_agent_deb-x64
```

Comment:
The job you need to depend on isn't called `heroku_agent_deb-x64` but `agent_heroku_deb-x64-a7` (see [this file](https://github.com/DataDog/datadog-agent/blob/bb8658b40cfd5f4ec3334eb23b1159928a31adf8/.gitlab/package_build/deb.yml#L240) with all deb package build job definitions).